README.dev.md

```diff
@@ -0,0 +1,149 @@
+If you forget the admin password, you can add a new admin account and then go to the sysadmin panel to reset user password.
+
+#### Domain
+
+You can create `/shared/bootstrap.conf`, write the following lines.
+
+    [server]
+    server.hostname = seafile.example.com
+
+And then restart service, nginx will update up config.
+
+#### Modify configurations
+
+The config files are under `shared/seafile/conf`. You can modify the configurations according to [Seafile manual](https://manual.seafile.com/)
+
+After modification, run the new docker container:
+
+```
+docker rm seafile-server
+docker run -d --name seafile-server -v /root/seafile:/shared -p 80:80 seafileltd/seafile:6.2.1
+```
+
+#### Find logs
+
+The seafile logs are under `shared/logs/seafile`.
+
+The system logs are under `shared/logs/var-log`.
+
+#### Add a new Admin
+
+Enter the command below.
+
+```
+docker exec -it seafile-server /opt/seafile/seafile-server-latest/reset-admin.sh
+```
+
+Enter the username and password according to the prompts.You now have a new admin account.
+
+### Directory Structure
+
+#### `/bootstrap`
+
+This directory is for container definitions for your Seafile containers. You are in charge of this directory, it ships empty.
+
+#### `/samples`
+
+Sample container definitions you may use to bootstrap your environment. You can copy templates from here into the bootstrap directory.
+
+#### `/shared`
+
+Placeholder spot for shared volumes. You may elect to store certain persistent information outside of a container, in our case we keep various logfiles and upload directory outside. This allows you to rebuild containers easily without losing important information.
+
+- /shared/db: This is the data directory for mysql server
+- /shared/seafile: This is the directory for seafile server configuration and data.
+- /shared/logs: This is the directory for logs.
+    - /shared/logs/var-log: This is the directory that would be mounted as `/var/log` inside the container. For example, you can find the nginx logs in `shared/logs/var-log/nginx/`.
+    - /shared/logs/seafile: This is the directory that would contain the log files of seafile server processes. For example, you can find seaf-server logs in `shared/logs/seafile/seafile.log`.
+- /shared/ssl: This is directory for certificate, which does not exist by default.
+- /shared/bootstrap.conf: This file does not exist by default. You can create it by your self, and write the configuration of files similar to the `samples` folder.
+
+#### `/templates`
+
+Various jinja2 templates used for seafile server configuration.
+
+#### `/image`
+
+Dockerfiles for Seafile.
+
+The Docker repository will always contain the latest built version at: https://hub.docker.com/r/seafileltd/seafile/, you should not need to build the base image.
+
+### Container Configuration
+
+#### Let's encrypt SSL certificate
+
+If you set `server.letsencrypt` to `true`, the bootstrap script would request a letsencrypt-signed SSL certificate for you.
```

Comment:
What about letting TLS be done by some other docker image? E.g. https://hub.docker.com/r/jrcs/letsencrypt-nginx-proxy-companion/